automerge/tests/test.rs

```diff
@@ -8,11 +8,11 @@ use helpers::{
 };
 #[test]
 fn no_conflict_on_repeated_assignment() {
-    let mut doc = Automerge::new();
+    let mut doc = AutoTxn::new();
     doc.set(&automerge::ROOT, "foo", 1).unwrap();
     doc.set(&automerge::ROOT, "foo", 2).unwrap();
     assert_doc!(
-        &doc,
+        doc.document(),
```

Comment:
Rather than calling `doc.document()` on every test I think it might be better to modify the `assert_doc` macro to make this call.